src/locales/en.json

```diff
@@ -1,7 +1,7 @@
 {
   "app.EditEnvironmentLimitsForm.cloneAll.yesNoQuestion": "Do you really want to use these limits for all the tests of all runtime environments?",
-  "app.acceptSolution.accepted": "Accepted",
-  "app.acceptSolution.notAccepted": "Accept this solution",
+  "app.acceptSolution.accepted": "Remove grading mark",
+  "app.acceptSolution.notAccepted": "Mark for grading",
   "app.addLicence.addLicenceTitle": "Add new licence",
```

Comment:
I would suggest "Accept as Final" and "Accepted as Final" for English translation. The "grading mark" is completely off.